message LHP_Blob {
  required uint32 color       = 1;
  required float  x1          = 2;
  required float  y1          = 3;
  required float  x2          = 4;
  required float  y2          = 5;
  optional float  cen_x       = 6;
  optional float  cen_y       = 7;
  optional uint32 bb_area     = 8;
  optional uint32 area        = 9;
}

message LHP_Frame {
  required uint32             frame_number  = 1;
  required uint32             camera_id     = 4;
  repeated LHP_Blob           blob         = 5;
}
